README.md

```diff
@@ -66,434 +66,6 @@ This roster is seeded with approvers from the Client workgroups.
 
 ---
 
-# Instructions
```

Comment:
Since we're making changes here, we may want to drop @julz and @ZhiminXiang from the serving roster

Reply:
done and also added myself as a volunteer, (can I just add myself?)

Reply:
> (can I just add myself?)

Yeah